person.rb

```diff
@@ -19,3 +26,45 @@ def of_age?
     @age >= 18
   end
 end
+# capitalizeDecorator.rb
+
+class CapitalizeDecorator < Nameable
+  def initialize(nameable)
+    @nameable = nameable
+    super()
+  end
+
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
+
+class Decorator < Nameable
+  def initialize(nameable)
+    @nameable = nameable
+    super()
+  end
+
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
```

Comment:
- Good job done so far, but your correct_name method should return `@nameable.correct_name`. Please fix this 🙏